### DISCUSSION TITLE
my ipad can not open the app

### DISCUSSION THREAD

**Discussion by hjx90609** [obiwan007/despresso#126] [Q&A]
My iPad mini2 can not open the despresso app and it is always crashing. The software version of my iPad is 12.5.7. I don't know if it's a compatibility issue, will it be improved in the future?

**Comment by obiwan007** [comment]
I can. Not really test it with such old version of ios,. Can't you update to a newer one?
Please uninstall and reinstall and make sure that you enable the location ser ices. Otherwise Bluetooth will not work.

**Reply by hjx90609** [reply]
That's a pity. 12.5.7 is the highest version this iPad can be updated to. I really like this app, and I saw that the introduction said that Apple devices above iOS 11 were supported, so I went to the second-hand market to buy this iPad. Can you find a way to fix it? Much appreciated!

**Reply by hjx90609** [reply]
https://github.com/obiwan007/despresso/assets/116875288/2e20310c-3c4c-4bb0-9dff-6e63d8fb46db

![mmexport1684425143647](https://github.com/obiwan007/despresso/assets/116875288/159f1d30-9ec5-4e90-8dda-fe7671c7f1d0)

**Reply by obiwan007** [reply]
Have you tried uninstalling? Is it asking for permissions? I do not have a way on testing it on such a device... So it is probably some API which is not supported anymore. In the future we will nail the app to support only the latest os version in the future to prevent such issues...
